pkg/inject/secrets-provider.go

```diff
@@ -6,30 +6,54 @@ import (
 )
 
 type SecretsProviderSidecarConfig struct {
-	containerMode           string
-	containerName           string
-	sidecarImage            string
-	secretsDestination      string
+	containerMode      string
+	containerName      string
+	sidecarImage       string
+	secretsDestination string
 }
-var validEnvVars = []string{
+
+var conjurEnvVars = []string{
 	"CONJUR_ACCOUNT",
+	"conjurAccount",
```

Comment:
Again, this is sufficient for now, but eventually we should update the Golden ConfigMap keys to their `ALL_CAPS` counterparts.